mne_neuron/dipole.py

```diff
@@ -101,18 +156,56 @@ def prsimtime():
                      np.array(dp_rec_L2.to_python()),
                      np.array(dp_rec_L5.to_python())]
 
-    pc.gid_clear()
-    pc.done()
 
     dpl = Dipole(np.array(t_vec.to_python()), dpl_data)
 
+    err = None
     if rank == 0:
+        if net.params['save_dpl']:
+            dpl.write('rawdpl_%d.txt' % trial)
+
         dpl.baseline_renormalize(net.params)
         dpl.convert_fAm_to_nAm()
         dpl.scale(net.params['dipole_scalefctr'])
         dpl.smooth(net.params['dipole_smooth_win'] / h.dt)
 
-    return dpl
+        try:
+            if extdata.any():
```

Comment:
Zen of Python says:
```py
>>> python -c "import this" | sed -n 7L
Flat is better than nested.
```

what I proposed [here](https://github.com/jasmainak/mne-neuron/pull/48/files#r294106114) is going to help avoid this awkward `try` + `if` clause.